### PR TITLE
add a note on connection pooling to do docs (close #2860)

### LINKED ISSUE #2860: update digitalocean docs to add a note about connection pooling
ref: https://discordapp.com/channels/407792526867693568/586606982077874176/621144673129332748

> I am trying out connection pool feature of Digital Ocean. Thus after applying the connection data the server only receives 502 bad gateway issue. Does anybody set this up before?
```
graphql-engine_1  | {"internal":{"statement":"BEGIN ISOLATION LEVEL SERIALIZABLE ","prepared":true,"error":{"exec_status":"FatalError","hint":null,"message":"prepared statement \"0\" already exists","status_code":"42P05","description":null},"arguments":[]},"path":"$","error":"postgres tx error","code":"postgres-error"}
```
> Oh I just realized that connection pooling is built-in at hasura: https://github.com/hasura/graphql-engine/issues/2491 So maybe a little side note at the DO docs (https://docs.hasura.io/1.0/graphql/manual/guides/deployment/digital-ocean-one-click.html#using-digitalocean-managed-postgres-database) would be helpful for users to make sure they use `session` mode.

### PR DESCRIPTION
**Pull request by santoshyadav198613**
fix #2860

### Affected components 

- Docs

### CODE REVIEW

**Comment by santoshyadav198613** [comment]
Hi @marionschleifer,
Please review and suggest the changes.

File: docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst [hunk 1]
```diff
@@ -301,6 +301,16 @@ database.
   to configure all required permissions (not applicable with the default
   connection string with DO Managed Postgres).
 
+Connection Pooling
+------------------
+
+  Connection Pooling is built-in feature of graphql-engine. The default connection pool size is 50.
```
Comment (marionschleifer) [inline docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst]: ...is **a** built-in...

File: docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst [hunk 2]
```diff
@@ -301,6 +301,16 @@ database.
...
+  If you need to configure pool size or timout you can use below environment variables.
```
Comment (marionschleifer) [inline docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst]: If you need to configure **the** pool size or **the** timeout, you can use **the** below environment variables.

File: docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst (hunk 1 above)
Comment (santoshyadav198613) [reply docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst]: Done.

File: docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst (hunk 2 above)
Comment (santoshyadav198613) [reply docs/graphql/manual/guides/deployment/digital-ocean-one-click.rst]: Done.

**Pull request merged**